src/ims/element/incident/incidents/_incidents.py

```diff
@@ -72,3 +72,12 @@ async def concentric_street_name_by_id(
         """
         namesByID = await self.config.store.concentricStreets(self.event.id)
         return jsonTextFromObject(namesByID)
+
+    @renderer
+    async def incident_types(
+        self, request: IRequest, tag: Tag
+    ) -> KleinRenderable:
+        types = await self.config.store.incidentTypes()
+        types = sorted(t for t in types)
+        types = tuple(t for t in types)
```

Comment:
Is this step needed? `jsonTextFromObject` should be able to ingest a `list`.

Reply:
https://github.com/burningmantech/ranger-ims-server/pull/1409